theme.json

```diff
@@ -0,0 +1,413 @@
+              "fontSize": "var(--wp--custom--heading--4--font-size)",
+              "lineHeight": "var(--wp--custom--heading--4--line-height)",
+              "fontWeight": "var(--wp--custom--heading--4--font-weight)",
+              "fontFamily": "var(--wp--custom--heading--4--font-family)",
+              "letterSpacing": "var(--wp--custom--heading--4--letter-spacing)"
+            }
+          },
+          "h3": {
+            "typography": {
+              "fontSize": "var(--wp--custom--heading--3--font-size)",
+              "lineHeight": "var(--wp--custom--heading--3--line-height)",
+              "fontWeight": "var(--wp--custom--heading--3--font-weight)",
+              "fontFamily": "var(--wp--custom--heading--3--font-family)",
+              "letterSpacing": "var(--wp--custom--heading--3--letter-spacing)"
+            }
+          },
+          "h4": {
+            "typography": {
+              "fontSize": "var(--wp--custom--heading--2--font-size)",
+              "lineHeight": "var(--wp--custom--heading--2--line-height)",
+              "fontWeight": "var(--wp--custom--heading--2--font-weight)",
+              "fontFamily": "var(--wp--custom--heading--2--font-family)",
+              "letterSpacing": "var(--wp--custom--heading--2--letter-spacing)"
+            }
+          },
+          "h5": {
+            "typography": {
+              "fontSize": "var(--wp--custom--heading--1--font-size)",
+              "lineHeight": "var(--wp--custom--heading--1--line-height)",
+              "fontWeight": "var(--wp--custom--heading--1--font-weight)",
+              "fontFamily": "var(--wp--custom--heading--1--font-family)",
+              "letterSpacing": "var(--wp--custom--heading--1--letter-spacing)"
+            }
+          },
+          "h6": {
+            "typography": {
+              "fontSize": "var(--wp--custom--heading--1--font-size)",
+              "lineHeight": "var(--wp--custom--heading--1--line-height)",
+              "fontWeight": "var(--wp--custom--heading--1--font-weight)",
+              "fontFamily": "var(--wp--custom--heading--1--font-family)",
+              "letterSpacing": "var(--wp--custom--heading--1--letter-spacing)"
+            }
+          }
+        }
+      },
+      "core/post-title": {
+        "color": {
+          "text": "var(--wp--custom--text--primary)"
+        },
+        "typography": {
+          "fontSize": "var(--wp--custom--heading--5--font-size)",
+          "lineHeight": "var(--wp--custom--heading--5--line-height)",
+          "fontWeight": "var(--wp--custom--heading--5--font-weight)",
+          "fontFamily": "var(--wp--custom--heading--5--font-family)",
+          "letterSpacing": "var(--wp--custom--heading--5--letter-spacing)"
+        }
+      }
+    }
+  },
+  "templateParts": [
+    {
+      "name": "header",
+      "title": "Header",
+      "area": "header"
+    },
+    {
+      "name": "footer",
+      "title": "Footer",
+      "area": "footer"
+    }
+  ],
+  "customTemplates": [
```

Comment:
Might be good to pick what WP default templates we want to include here. Everything falls back in a smart way but we should probably include a basic set to account for core usage.

Reply:
Hmm, do you mean including things like home, 404, search, etc?